galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.cemt.manager.ivt/src/main/java/dev/galasa/cicsts/cemt/manager/ivt/CEMTManagerIVT.java

```diff
@@ -50,19 +51,20 @@ public void login() throws Exception {
       
       resourceTerminal.clear();
       resourceTerminal.waitForKeyboard(); 
+      
    }
    
    @Before
    public void installProgramResource() throws CicstsManagerException, TimeoutException, KeyboardLockedException, TerminalInterruptedException, NetworkException, FieldNotFoundException {
-      resourceTerminal.type("CEDA DEFINE PROGRAM(EXAMPLE) GROUP(EXGROUP)").enter().waitForKeyboard();
+	  resourceTerminal.type("CEDA DEFINE PROGRAM(EXAMPLE) GROUP(EXGROUP)").enter().waitForKeyboard();
       resourceTerminal.pf3().waitForKeyboard().clear().waitForKeyboard();
       resourceTerminal.type("CEDA INSTALL PROGRAM(EXAMPLE) GROUP(EXGROUP)").enter().waitForKeyboard();
       resourceTerminal.pf3().waitForKeyboard().clear().waitForKeyboard();  
    }
    
    @After
    public void clearResources() throws KeyboardLockedException, NetworkException, TerminalInterruptedException, TimeoutException, FieldNotFoundException, InterruptedException {
```

Comment:
Be careful - another test might be trying to use this program name

Reply:
Same as my previous comment - could mitigate this using random strings for resource names so there are no clashes.